db/migrate/20161215230219_add_invoice_text_and_invoice_logo_to_enterprises.rb

```diff
@@ -0,0 +1,6 @@
+class AddInvoiceTextAndInvoiceLogoToEnterprises < ActiveRecord::Migration
+  def change
+    add_column :enterprises, :invoice_text, :text
+    add_column :enterprises, :invoice_logo, :boolean, default: false
```

Comment:
Perhaps `display_invoice_logo` would give a better indication of what this field does? Then rails magic will allow us to do `enterprise.display_invoice_logo?`, which is a bit more readable. I thought this was a new image field when I first read it in the code.